packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -51,12 +50,9 @@ export class ColorPickerPopover<T extends CustomPickersDescriptor> extends React
   };
 
   handleChange = (color: any) => {
-    const { onColorChange, onChange, enableNamedColors, theme } = this.props;
+    const { onColorChange, onChange, theme } = this.props;
     const changeHandler = onColorChange || onChange;
 
-    if (enableNamedColors) {
-      return changeHandler(color);
-    }
     changeHandler(getColorForTheme(color, theme));
```

Comment:
this is the path that can go away

Reply:
done! 🥳

Reply:
@thisisobate seems like you have not updated this yet - let us know when it's updated. It should be `changeHandler(color)`